WORKSPACE

```diff
@@ -231,9 +231,11 @@ dist_http_file(
     downloaded_file_path = "zulu-win-vanilla.zip",
 )
 
-dist_http_file(
+http_file(
```

Comment:
Can we keep using dist_http_file and move the definition into distdir_deps.bzl?

Reply:
Done !